src/content/en/fundamentals/getting-started/service-worker/learn-more-get-help.markdown

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
wrap this doc into the rough edges doc, feels odd to have it as a standalone. but then again, maybe not. will leave to your judgement.